doc/getting_started/installation.md

```diff
@@ -78,6 +78,38 @@ Make sure Panel is installed in the same environment as JupyterLab/Jupyter Noteb
 If you plan to use Panel in a non-Jupyter notebook environment, such as Google Colab or VSCode, refer to the [relevant how-to section](../how_to/notebook/other_nb.md).
 :::
 
+## Updating Panel
```

Comment:
Maybe this should be somewhere else? But I don't know where.

Reply:
I included it because 1) This user had an old version of panel in his python environment because its a dependency in the project in was handed our from another colleague. 2) updating panel is not straightforward and should be documented.

Reply:
Definitely don't think it should be here.

Reply:
Ok. Removed it.